docs/src/user/cli/info.rst

```diff
@@ -59,3 +65,31 @@ Here is an example of all the sections provided by the command.
 
 The last section contains information about the best trial so far, providing its
 hyperparameter values and the corresponding objective.
+
+The ``--name`` argument
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Unless the command has been changed this should be removed.

Reply:
It was changed.